### PR TITLE
Add first batch of sponsors logos

### PR DESCRIPTION
**Pull request by modkaffes**
Addresses #146.
Please keep in mind that the issue is not resolved yet, we're waiting on Microsoft's official logo and url, as I have currently put something temporary.

### CODE REVIEW

File: app/_includes/sponsors.html [hunk 1]
```diff
@@ -21,9 +21,9 @@
   </div>
   <div class="row">
     <div class="partners premium-partners">
-      <div class="col-md-offset-4 col-sm-6 col-md-4 partner">
-        <a href="http://www.amdtelecom.net/" target="_blank">
-          <img src="images/sponsors/amd-telecom_logo.jpg" alt="AMD Telecom" class="img-responsive">
+      <div class="col-sm-6col-md-12 partner">
```
Comment (kbariotis) [inline app/_includes/sponsors.html]: is that one word?

Comment (kbariotis) [inline app/_includes/sponsors.html]: @modkaffes You also need to edit the Site's menu to include the Sponsors section.

**Comment by thanpolas** [comment]
Let's not have MS block us, remove it for now and when I have the info i'll open an new issue

**Comment by modkaffes** [comment]
@thanpolas OK.
@kbariotis on it.

File: app/_includes/sponsors.html (hunk 1 above)
Comment (vkarampinis) [inline app/_includes/sponsors.html]: @modkaffes fix this typo and will be ok for merge!

**Pull request merged**